src/consensus/aft/test/main.cpp

```diff
@@ -1096,4 +1096,47 @@ DOCTEST_TEST_CASE("Exceed append entries limit")
     (sent_entries > num_small_entries_sent &&
      sent_entries <= num_small_entries_sent + num_big_entries));
   DOCTEST_REQUIRE(r2.ledger->ledger.size() == individual_entries);
+}
+
+DOCTEST_TEST_CASE("Test Asynchronous Execution Coordinator")
```

Comment:
- first tx with false as its sync
- use primary to create sequence
  - no shuffle
  - shuffle but happy exec 
- on primary to create order -  shuffle - see conflict occurred